CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Lightningd: add support for `signet` networks using the `--network=signet` or `--signet` startup option
 - JSON API: `listfunds` now returns also `funding_output` for `channels`
+- Plugin: new notification `forward_event` to report `FORWARD_OFFERED`, `FORWARD_SETTLED`, `FORWARD_FAILED` and `FORWARD_LOCAL_FAILED` events for forward payments.
```

Comment:
Uses old names? But really, CHANGELOG should aim for  one line, so "Plugin: new notification `forward_event` offered/settled/failed/local_failed" is sufficient.

Reply:
Thank you, it's better description. Corrected.